manager/controlapi/service.go

```diff
@@ -334,7 +334,7 @@ func (s *Server) ListServices(ctx context.Context, request *api.ListServicesRequ
 	if request.Filters != nil {
 		services = filterServices(services,
 			func(e *api.Service) bool {
-				return filterContains(e.Spec.Annotations.Name, request.Filters.Names)
+				return filterContainsPrefix(e.Spec.Annotations.Name, request.Filters.Names)
```

Comment:
Thanks @stevvooe @aaronlehmann. I missed that one in the pull request. Let me update the pull request shortly.